Makefile

```diff
@@ -131,7 +131,10 @@ test-python-ci:
 		--ignore tests/sentry/search/events \
 		--ignore tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
 		--ignore tests/sentry/region_to_control/test_region_to_control_kafka.py \
-		--cov . --cov-report="xml:.artifacts/python.coverage.xml"
+		--cov . --cov-report="xml:.artifacts/python.coverage.xml" \
+		--durations=0 \
```

Comment:
Not sure if you meant to keep this in, but if we we have this flag defined here: https://github.com/getsentry/sentry/blob/master/.github/actions/setup-sentry/action.yml#L77

Reply:
We're debugging some stuff at the moment